File: src/test/java/com/dsb/rest/model/StatisticTest.java
package com.dsb.rest.model;

import org.junit.Assert;
import org.junit.Test;

public class StatisticTest {

    @Test
    public void hasForceTrue() {
        FromTo fromTo = new FromTo();
        fromTo.setFrom("1");
        fromTo.setTo("1");

        Statistic statistic = new Statistic();
        statistic.setForce(fromTo);
        Assert.assertEquals(statistic.hasForce(), true);
    }

    @Test
    public void hasForceFalse() {
        Statistic statistic = new Statistic();
        Assert.assertEquals(statistic.hasForce(), false);
    }
}

File: src/test/kotlin/com/dsb/rest/model/StatisticTest.kt
package com.dsb.rest.model

import org.junit.Assert
import org.junit.Test

class StatisticTest {
    @Test
    fun hasForceTrue() {
        val fromTo = FromTo()
        fromTo.min = "1"
        fromTo.max = "1"
        val statistic = Statistic()
        statistic.force = fromTo
        Assert.assertEquals(statistic.hasForce(), true)
    }

    @Test
    fun hasForceFalse() {
        val statistic = Statistic()
        Assert.assertEquals(statistic.hasForce(), false)
    }
}